prow/kube/prowjob.go

```diff
@@ -101,8 +101,24 @@ type ProwJobStatus struct {
 	State          ProwJobState `json:"state,omitempty"`
 	Description    string       `json:"description,omitempty"`
 	URL            string       `json:"url,omitempty"`
-	PodName        string       `json:"pod_name,omitempty"`
-	BuildID        string       `json:"build_id,omitempty"`
+
+	// PodName applies only to ProwJobs fulfilled by
+	// plank. This field should always be the same as
+	// the ProwJob.ObjectMeta.Name field.
+	PodName string `json:"pod_name,omitempty"`
+
+	// ProwJobID is the build identifier vended by `tot`
```

Comment:
This is not only vended by tot; there is also the option to use the snowflake library.